test_factorial (__main__.TestExercice) ... FAIL
test_prefixes (__main__.TestExercice) ... ERROR
test_summation (__main__.TestExercice) ... FAIL

======================================================================
ERROR: test_prefixes (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 14, in test_prefixes
    output = exercice.use_prefixes()
  File "/home/runner/work/c03_ch5_exercices-JordanLow682/c03_ch5_exercices-JordanLow682/exercice.py", line 22, in use_prefixes
    nom = print(prefixes[index] + suffixes)
TypeError: string indices must be integers

======================================================================
FAIL: test_factorial (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 50, in test_factorial
    'Erreur dans le remplacement de caractère'
AssertionError: Lists differ: [0, 0, 0, 0] != [1, 6, 40320, 3628800]

First differing element 0:
0
1

- [0, 0, 0, 0]
+ [1, 6, 40320, 3628800] : Erreur dans le remplacement de caractère

======================================================================
FAIL: test_summation (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 38, in test_summation
    'Retrait du mauvais caractère'
AssertionError: 0 != 24133 : Retrait du mauvais caractère

----------------------------------------------------------------------
Ran 3 tests in 0.001s

FAILED (failures=2, errors=1)
